File: 462-minimum-moves-to-equal-array-elements-ii/462-minimum-moves-to-equal-array-elements-ii.cpp
class Solution {
public:
    int minMoves2(vector<int>& v1) {
        sort(begin(v1),end(v1));
        int l1 = v1[ v1.size()>>1];
        int ans1 = 0;
        for(auto &it:v1){
            ans1 += abs(it-l1);
        }
        return ans1;
    }
};

File: 462-minimum-moves-to-equal-array-elements-ii/462-minimum-moves-to-equal-array-elements-ii.py
class Solution:
    def minMoves2(self, v1: List[int]) -> int:
        v1.sort();
        l1 = v1[len(v1)>>1]
        ans1 = 0
        for i in v1:
            if l1>i:
                ans1 += l1-i
            else:
                ans1 += i-l1
        return ans1